README.md

```diff
@@ -50,20 +50,17 @@ in the napari plugin [napari-segment-blobs-and-things-with-membranes](https://ww
 
 ### Measurements
 The first step is deriving measurements from the labelled image and the corresponding pixels in the grey-value image.
-You can use the menu `Tools > Measurement > Measure intensity, shape and neighbor counts (ncp)` for that.
+You can use the menu `Tools > Measurement > Label statistics (clEsperanto)` which is included in the [napari-pyclesperanto-assistant](https://www.napari-hub.org/plugins/napari-pyclesperanto-assistant)
```

Comment:
Silly question, but any idea why can't I see this entry in the tools menu? I updated the assistant and restarted napari, but I can only do measurements from the UI of the assistant and not the tools menu 🤔
![image](https://user-images.githubusercontent.com/52177660/198005995-ae56065c-31b9-462f-9245-3c4c26e0d509.png)

Reply:
@lazigu 
You need to have the pyclesperanto assistant installed not just the napari assistant. Maybe we should make it more obvious in the readme? Although we do link the repository you need to install here... We could alternatively mention the scikit regionprops as well as the pyclesperanto regionprops as the scikit regionprops come preinstalled with nsc?

Reply:
Hmm, which one do you mean? I have pyclesperanto-prototype installed if you are talking about this one. Now I updated it, but such a menu still hasn't appeared. Sorry, I might be missing something very obvious, but my brain is still in a holiday mood :D I might try a fresh environment, because now I was using an old one

Reply:
hahahahaha no problem at all. I mean the pyclesperanto assistant which is the new link in the readme (https://www.napari-hub.org/plugins/napari-pyclesperanto-assistant) This is basically a library for all pyclesperanto functions in napari and was the precursor to the napari assistant. Nevertheless, if you are confused, then so will be other users. I will modify the Readme slightly so just tell me what you think once I've made the changes :)

Reply:
Ahhhh right, I had just napari-assistant in that environment :D I knew it was something silly I was missing..
But yes I think maybe it would be nice to mention it clearer, and also to mention non-gpu version of deriving measurements